### DISCUSSION TITLE
When elysia integrate with sveltekit, is the performance same as pure elysiajs?

### DISCUSSION THREAD

**Discussion by FarizInk** [elysiajs/elysia#427] [Q&A] [answered]
last night I was reading [this](https://elysiajs.com/integrations/sveltekit.html) and trying, I have question that elysia performance same or reduce? because sveltekit combined with elysia, thank you 🙂

**Comment by SaltyAom** [comment, marked as answer]
There is probably a slight performance decrease likely 1-2% from my guess if you are using Bun.

**Reply by FarizInk** [reply]
okay, thank you
